listen/property-changes.js

```diff
@@ -70,8 +68,9 @@ PropertyChanges.prototype.getOwnPropertyChangeDescriptor = function (key) {
         });
     }
     var objectPropertyChangeDescriptors = this.__propertyChangeListeners__;
-    if (!object_owns.call(objectPropertyChangeDescriptors, key)) {
+    if (!objectPropertyChangeDescriptors.hasOwnProperty(key)) {
```

Comment:
@kriskowal If instead of {} I use Object.create(null) and then I do my checkings with: if (!objectPropertyChangeDescriptors[key]) {...}, do you see any downside?